core/templates/dev/head/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.mainpage.html

```diff
@@ -25,6 +25,6 @@
     </div>
 
     <script src="/third_party/static/ckeditor-4.12.1/ckeditor.js"></script>
-    @require('../footer_js_libs.html')
+    @load('/pages/footer_js_libs.html')
```

Comment:
Please modify the `load` function to work without the starting `/` so instead of this you would have `@load('pages/footer_js_libs.html')`.

Reply:
Done